qdrant-landing/content/documentation/Tutorials & Examples/cloud-inference-hybrid-search.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Using Cloud Inference with Qdrant for Vector Search
+weight: 35
+---
+# Using Cloud Inference with Qdrant for Vector Search
+## Install Qdrant Client
+```bash
+pip install qdrant-client
+```
+## Initialize the Client
+Initialize the Qdrant client after creating a [Qdrant Cloud account](/documentation/cloud/) and a [dedicated paid cluster](/documentation/cloud/create-cluster/). 
```

Comment:
It makes sense to mention the importance of setting `cloud_inference=True` here. Otherwise, it might get lost.